src/server/Listener.js

```diff
@@ -43,20 +43,25 @@ class Listener {
 	add (options) {
 		const {uri, method} = options;
 		this.mocks[uri] = this.mocks[uri] || {};
-
-		// Register mock if it hasn't been registered before
-		if (!this.mocks[uri][method]) {
-			this.app[method.toLowerCase()](uri, (req, res) => {
-				return this.mocks[uri][method].handler(req, res);
-			});
+		const oldMock = this.mocks[uri][method];
```

Comment:
I feel it would be better to call it `currentMock`. Then if there is currentMock, then override it. What do you think?